wagtailio/blog/templates/blog/blog_index_page.html

```diff
@@ -5,23 +5,24 @@
 {% block body_class %}template-blog{% endblock %}
 
 {% block content %}
-    <div class="page-content">
-        <h1>Blog</h1>
 
-        <ul class="blog-listing">
-            {% for post in posts %}
-                <li class="blog-listing__item">
-                    <div class="date"><span>{{ post.date|date }}</span></div>
-                    <h2 class="blog-listing__title"><a href="{{ post.url }}">{{ post.title }}</a></h2>
-
-                    <p>{{ post.introduction }}</p>
+    <div class="container">
+        <div class="hero">
+            <div class="hero__inner">
+                <h1 aria-hidden="true">Latest Chirps</h1>
+                <span class="u-sr-only">Latest blog posts</span>
+            </div>
+        </div>
 
-                    {% if post.author %}
-                        {% include "blog/includes/post_author.html" with author=post.author only %}
-                    {% endif %}
-                </li>
+        <section class="blog-listing">
+            {% for post in posts %}
+                {% if forloop.first or forloop.counter|divisibleby:6 %}
```

Comment:
I think the featured posts may end up being a curated list so this logic will change, but we can merge as is as we have the separation of the featured post from a normal post.